.github/workflows/cypress-ete.yml

```diff
@@ -24,7 +24,8 @@ jobs:
           CYPRESS_SIGN_IN_PAGE_URL: https://profile.code.dev-theguardian.com/signin
           IS_HTTPS: false
           APP_SECRET: ${{ secrets.APP_SECRET }}
-          CYPRESS_MAILSLURP_KEY: ${{ secrets.CYPRESS_MAILSLURP_KEY }}
+          CYPRESS_MAILOSAUR_API_KEY: ${{ secrets.CYPRESS_MAILOSAUR_API_KEY }}
+          CYPRESS_MAILOSAUR_SERVER_ID: ${{ secrets.CYPRESS_MAILOSAUR_SERVER_ID }}
```

Comment:
@coldlink  and I had a chat and decided we should make this a secret too, just in case someone comes along and decides to send random emails to our account.

I forgot to push a change to the shell script & the docs to make sure people know to set this locally. Will push something next week (hackday again tomorrow)